raw_data/preprocessed/africa/niger_french.txt

```diff
@@ -0,0 +1,261 @@
errestres, marins et autres écosystèmes aquatiques et les complexes écologiques dont ils font partie: cela comprend la diversité au sein des espèces et entre espèces ainsi que celles des écosystèmes; Écosystème: le complexe dynamique formé de communautés de plantes d’animaux et de micro-organismes et de leur environnement non vivant qui par leurs interactions forment une unité fonctionnelle; Effluent: tout rejet liquide ou gazeux d’origine domestique, agricole et industrielle, traité ou non traité et déversé directement ou indirectement dans la nature; Environnement: l’ensemble des éléments physiques, chimiques et biologiques, des facteurs sociaux et des relations dynamiques entretenues entre ces différentes composantes; Équilibre écologique: le rapport relativement stable créé progressivement au cours des temps entre l’homme, la faune et la flore, ainsi que leur interaction avec les conditions du milieu naturel dans lequel ils vivent; Établissements classés: les établissements qui présentent des causes de danger ou des inconvénients, soit pour la sécurité, la salubrité ou la commodité du voisinage ou pour la santé publique, soit encore pour l’agriculture; Établissement humain: l’ensemble des agglomérations urbaines et rurales quels que soient leur type et leur taille, et l’ensemble des infrastructures dont elles doivent disposer pour assurer à leurs habitants une existence saine et équilibrée; Étude d’impact sur l’environnement: le rapport d’évaluation des changements négatifs ou positifs que la réalisation d’une activité, d’un projet, d’un programme ou d’un plan de développement risque de causer à l’environnement; Installation: toute source fixe susceptible d’être génératrice d’atteinte à l’environnement quels que soient son propriétaire ou sa destination; Nuisance: toute agression d’origine humaine contre le milieu physique biologique, naturel ou artificiel entourant l’homme et causant un simple désagrément ou un véritable dommage à ce denier; Polluant: tout rejet solide, liquide ou gazeux, tout déchet, odeur, chaleur, son, vibration, rayonnement ou combinaison de ceux-ci susceptible de provoquer une pollution; Pollueur: toute personne physique ou morale émettant un polluant qui entraîne une atteinte à l’environnement; Pollution: toute contamination ou modification directe ou indirecte de l’environnement provoquée par un acte susceptible: – d’influer négativement sur le milieu; – de provoquer une situation préjudiciable pour la santé, la sécurité, le bien-être de l’homme, de la flore et de la faune ou les biens collectifs et individuels; Ressources génétiques: l’élément des ressources biologiques d’origine végétale, animale, microbienne ou autre, contenant des unités fonctionnelles de l’hérédité et ayant une valeur effective ou potentielle pour l’humanité; Ressources naturelles: l’ensemble des produits naturels, des écosystèmes, des éléments abiotiques et des équilibres qui composent la terre ainsi que des diverses formes d’énergie naturelles.
```

Comment:
1. `ce denier; Polluant` should instead read `ce dernier; Polluant`.
2. The bullet-point-like dashes in the definition of Pollution should be replaced with enclosed bullet points.